### PR TITLE
Add 1.15.3 release notes

### PR DESCRIPTION
**Pull request by Monkeyanator**
Required before https://github.com/istio/release-builder/pull/1256 merge

### CODE REVIEW

File: content/en/news/releases/1.15.x/announcing-1.15.3/index.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: Announcing Istio 1.15.3
+linktitle: 1.15.3
+subtitle: Patch Release
+description: Istio 1.15.3 patch release.
+publishdate: 2022-10-26
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15.3/index.md:6]: Haven't looked, but did we release this today?

File: content/en/news/releases/1.15.x/announcing-1.15.3/index.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    - /news/announcing-1.15.3
+---
+
+This release contains bug fixes to improve robustness. This release note describes what’s different between Istio 1.15.2 and Istio 1.15.3.
+
+{{< relnote >}}
+
+## Changes
+
+- **Updated** the default value for `TRUSTED_GATEWAY_CIDR`. Previously, this was empty, which caused the XFCC authenticator to reject non-loopback requests.
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15.3/index.md:18]:
```suggestion
- **Updated** the default value for `TRUSTED_GATEWAY_CIDR`. Previously this was empty, which caused the XFCC authenticator to reject non-loopback requests.
```

File: content/en/news/releases/1.15.x/announcing-1.15.3/index.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- **Added** validation warnings when a `DestinationRule` specifies failover policies but does not provide an outlier detection policy. Previously, istiod was silently ignoring the failover settings.
+
+- **Fixed** an issue causing `kube-inject` to crash when the pod annotation `proxy.istio.io/config` is set.
+
+- **Fixed** an issue with missing `service_name` in Telemetry API when configuring Datadog tracing provider. ([Issue #38573](https://github.com/istio/istio/issues/38573))
```
Comment (ericvn) [inline content/en/news/releases/1.15.x/announcing-1.15.3/index.md:24]: maybe 
```suggestion
- **Fixed** an issue with a missing `service_name` in the Telemetry API when configuring a Datadog tracing provider. ([Issue #38573](https://github.com/istio/istio/issues/38573))
```

File: content/en/news/releases/1.15.x/announcing-1.15.3/index.md (hunk 1 above)
Comment (Monkeyanator) [reply content/en/news/releases/1.15.x/announcing-1.15.3/index.md]: No did not release yesterday.

**Comment by dhawton** [comment]
Will manually cherry pick and update the data/args.yml after this is approved.

**Review by ericvn — APPROVED** [review]
Looks good. I'll approve and expect a manual cherry-pick for `release-1.15`. I plan to approve and DNM that PR so the RM can remove the label and merge when appropriate.

**Pull request merged**